lib/libzutil/zutil_pool.c

```diff
@@ -48,14 +67,14 @@ dump_ddt_stat(const ddt_stat_t *dds, int h)
 	else
 		zfs_nicenum(1ULL << h, refcnt, sizeof (refcnt));
 
-	zfs_nicenum(dds->dds_blocks, blocks, sizeof (blocks));
-	zfs_nicebytes(dds->dds_lsize, lsize, sizeof (lsize));
-	zfs_nicebytes(dds->dds_psize, psize, sizeof (psize));
-	zfs_nicebytes(dds->dds_dsize, dsize, sizeof (dsize));
-	zfs_nicenum(dds->dds_ref_blocks, ref_blocks, sizeof (ref_blocks));
-	zfs_nicebytes(dds->dds_ref_lsize, ref_lsize, sizeof (ref_lsize));
-	zfs_nicebytes(dds->dds_ref_psize, ref_psize, sizeof (ref_psize));
-	zfs_nicebytes(dds->dds_ref_dsize, ref_dsize, sizeof (ref_dsize));
+	fmt_num(parsable, dds->dds_blocks, blocks, sizeof (blocks));
+	fmt_bytes(parsable, dds->dds_lsize, lsize, sizeof (lsize));
+	fmt_bytes(parsable, dds->dds_psize, psize, sizeof (psize));
+	fmt_bytes(parsable, dds->dds_dsize, dsize, sizeof (dsize));
+	fmt_num(parsable, dds->dds_ref_blocks, ref_blocks, sizeof (ref_blocks));
+	fmt_bytes(parsable, dds->dds_ref_lsize, ref_lsize, sizeof (ref_lsize));
+	fmt_bytes(parsable, dds->dds_ref_psize, ref_psize, sizeof (ref_psize));
+	fmt_bytes(parsable, dds->dds_ref_dsize, ref_dsize, sizeof (ref_dsize));
```

Comment:
You should be able to use the existing `zfs_nicenum_format()` function here and either specify the `ZFS_NICENUM_BYTES` or `ZFS_NICENUM_RAW` for the formatting.